microsoft-edge/dualengine/concepts/adapter-dll.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Understanding the DualEngine Adapter DLL
+description: Understanding the requirements and uses of DualEngine Adapter DLL.
+author: MSEdgeTeam
+ms.author: edgededev
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.localizationpriority: high
+ms.technology: windows-integration
+ms.date: 11/06/2023
+---
+
+# Understanding the DualEngine Adapter DLL
+The DualEngine API is an API that can only be used from inside the Internet Explorer process.
+As such in order to make use of the API it is necessary to create a plugin DLL that Internet Explorer will load.

```

Comment:
```suggestion
To use this API, it's necessary to create a plugin DLL that Internet Explorer will load.
```
probably no net gain in clarity by keeping the extra words

Reply:
The DualEngine API can only be used from inside the Internet Explorer process.  Therefore, to use the DualEngine API, you must create a plugin DLL that Internet Explorer loads.  This DLL hosts your code that uses the DualEngine API, as well as any code that you need to communicate with your main application; that's why it's referred to as the _adapter_.